tensorboard/webapp/notification_center/_views/notification_center_container.ts

```diff
@@ -13,22 +13,33 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 import {Component} from '@angular/core';
-import {Notification} from '../_redux/notification_center_types';
+import {ViewNotificationExt} from './view_types';
+import {notificationNotes} from './notification_notes';
+import {from, Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {CategoryEnum} from '../_redux/notification_center_types';
```

Comment:
use VSCode's "Organize import". Specifically, `rxjs/operators` should appear above the file module imports.

Reply:
Done. Thanks!